app/pages/patientdata/patientdata.js

```diff
@@ -526,6 +512,61 @@ export let PatientData = React.createClass({
       datetimeLocation: datetime
     });
   },
+
+  componentWillMount: function() {
+    this.doFetching(this.props);
+    this.doProcessing(this.props);
```

Comment:
I think it was for the refresh, but you solved that in another way, so I think we can delete? If it doesn't break on delete, that is.